### PR TITLE
p5.js version of CC137

### PR DESCRIPTION
**Pull request by PARC6502**
Ported over CC137, I used [CCapture](https://github.com/spite/ccapture.js) for saving the frames, and I used points instead of the loadPixels() as I couldn't get that to display well on any of my browsers and worried others would have the same issue.

Here's the output I got from this p5js version:
open_simplex_noise_loop

### CODE REVIEW

**Comment by PARC6502** [comment]
Not really important to add this, but I was a little happier with this earlier output

output

File: CodingChallenges/CC_137_4D_Noise_Loop/P5/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>4D Open Simplex Noise Loop</title>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.2/p5.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.2/addons/p5.dom.min.js"></script>
+  <script src="OpenSimplexNoise.js"></script>
+  <script src="CCapture.all.min.js"></script>
```
Comment (shiffman) [inline CodingChallenges/CC_137_4D_Noise_Loop/P5/index.html]: Is there a CDN link for `CCapture`? I wasn't aware of this library, great option for recording!

**Comment by PARC6502** [comment]
Yeah, there is!

https://unpkg.com/ccapture.js@1.1.0/build/CCapture.all.min.js

**Comment by gruselhaus** [comment]
> Yeah, there is!
https://unpkg.com/ccapture.js@1.1.0/build/CCapture.all.min.js

@PARC6502 Would you mind to edit the code to load the Library via the CDN. I could make these changes if you want me to.

**Comment by PARC6502** [comment]
> @PARC6502 Would you mind to edit the code to load the Library via the CDN. I could make these changes if you want me to.

All done!

**Comment by gruselhaus** [comment]
Great ❤️ Thanks!

**Pull request merged**